README.md

```diff
@@ -85,6 +85,8 @@ end
 ```julia
 julia> using PortAudio, SampledSignals, LibSndFile
 
+julia> using FileIO: load, save, loadstreaming, savestreaming
+
 julia> stream = PortAudioStream("Built-in Microph", 2, 0)
```

Comment:
```suggestion
julia> stream = stream = PortAudioStream(1, 0) # default input (e.g., built-in microphone)
```
This version should work on any OS; the original version did not work on my Mac.